scripts/akrtma3d/exakrtma3d_rhist.ksh

```diff
@@ -45,105 +68,148 @@ rhcycle=t${rhcyc}z
+# 
+#cyclist=t25z
+#if [ $rhcyc = "00" ]
+#then
+#   cyclist="t00z|t01z|t02z|t03z|t04z|t05z"
+#   endcyc=05
+#elif [ $rhcyc = "06" ]
+#then
+#   cyclist="t06z|t07z|t08z|t09z|t10z|t11z"
+#   endcyc=11
+#elif [ $rhcyc = "12" ]
+#then
+#   cyclist="t12z|t13z|t14z|t15z|t16z|t17z"
+#   endcyc=17
+#elif [ $rhcyc = "18" ]
+##then
+ #  cyclist="t18z|t19z|t20z|t21z|t22z|t23z"
+ #  endcyc=23
+#fi
+
+   #Redefine cyclist to save single analysis files MPondeca/30Jul2017
    cyclist="${rhcyc}"                            #MPondeca/30Jul2017
 
 cd $DATA
 
-cp rtma3d.${cyclem1}.* $DATA/$PDYm1
-cp rtma3d.${cyclem2}.* $DATA/$PDYm2
-cp rtma3d.${cyclem3}.* $DATA/$PDYm3
+cp $COMINm1/rtma3d.${cyclem1}.* $DATA/$PDYm1
+cp $COMINm2/rtma3d.${cyclem2}.* $DATA/$PDYm2
+cp $COMINm3/rtma3d.${cyclem3}.* $DATA/$PDYm3
 
-ln -sf $COMINobsproc_rtma3dm1 $DATA/$PDYm1
-ln -sf $COMINobsproc_rtma3dm2 $DATA/$PDYm2
-ln -sf $COMINobsproc_rtma3dm3 $DATA/$PDYm3
+ln -sf  $COMINobsproc_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMINobsproc_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMINobsproc_rtma3dm3 $DATA/$PDYm3
 
-ln -sf $GESINhrrr_rtma3dm1 $DATA/$PDYm1
-ln -sf $GESINhrrr_rtma3dm2 $DATA/$PDYm2
-ln -sf $GESINhrrr_rtma3dm3 $DATA/$PDYm3
+ln -sf  $GESINhrrr_rtma3dm1 $DATA/$PDYm1
+ln -sf  $GESINhrrr_rtma3dm2 $DATA/$PDYm2
+ln -sf  $GESINhrrr_rtma3dm3 $DATA/$PDYm3
 
-ln -sf $COMOUTgsi_rtma3dm1 $DATA/$PDYm1
-ln -sf $COMOUTgsi_rtma3dm2 $DATA/$PDYm2
-ln -sf $COMOUTgsi_rtma3dm3 $DATA/$PDYm3
+ln -sf  $COMOUTgsi_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMOUTgsi_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMOUTgsi_rtma3dm3 $DATA/$PDYm3
 
-ln -sf $COMOUTpost_rtma3dm1 $DATA/$PDYm1
-ln -sf $COMOUTpost_rtma3dm2 $DATA/$PDYm2
-ln -sf $COMOUTpost_rtma3dm3 $DATA/$PDYm3
+ln -sf  $COMOUTpost_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMOUTpost_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMOUTpost_rtma3dm3 $DATA/$PDYm3
 
-ln -sf $COMOUTbufrsnd_rtma3dm1 $DATA/$PDYm1
-ln -sf $COMOUTbufrsnd_rtma3dm2 $DATA/$PDYm2
-ln -sf $COMOUTbufrsnd_rtma3dm3 $DATA/$PDYm3
+ln -sf  $COMOUTbufrsnd_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMOUTbufrsnd_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMOUTbufrsnd_rtma3dm3 $DATA/$PDYm3
 
-ln -sf $COMOUTautoqc_rtma3dm1 $DATA/$PDYm1
-ln -sf $COMOUTautoqc_rtma3dm2 $DATA/$PDYm2
-ln -sf $COMOUTautoqc_rtma3dm3 $DATA/$PDYm3
+ln -sf  $COMOUTautoqc_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMOUTautoqc_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMOUTautoqc_rtma3dm3 $DATA/$PDYm3
 
+ln -sf  $COMOUThrrrdas_rtma3dm1 $DATA/$PDYm1
+ln -sf  $COMOUThrrrdas_rtma3dm2 $DATA/$PDYm2
+ln -sf  $COMOUThrrrdas_rtma3dm3 $DATA/$PDYm3
 
-ls -1r */*/* | awk '
 
-            /prepbufr/ { print "./"$0 > "perm" ; next }
+ls -1r */* | awk '
 
-	    /bufr/ { print "./"$0 > "perm" ; next }
+            /rejectlist/ { print "./"$0 > "perm1" ; next }
 
-            /NASALaRCCloudInGSI/ { print "./"$0 > "perm" ; next }
+            /smarthrrrconus/ { print "./"$0 > "perm1" ; next }
 
-            /NSSLRefInGSI/ { print "./"$0 > "perm" ; next }
+	    /smarthrrrak/ { print "./"$0 > "perm1" ; next }
```

Comment:
These files do not exist - smarthrrrconus and smarthrrrak.  Why are you looking for these files?  Just search for /smart/.